pkg/process/utils.go

```diff
@@ -127,6 +127,7 @@ func checkKillError(err error) error {
 	}
 	if strings.Contains(err.Error(), "os: process already finished") ||
 		strings.Contains(err.Error(), "container not running") ||
+		strings.Contains(strings.ToLower(err.Error()), "no such process") ||
```

Comment:
Is this different from unix.ESRCH `{3, "ESRCH", "no such process"}` check 1 line below? ([link](https://github.com/golang/sys/blob/master/unix/zerrors_linux_amd64.go#L2954))

Reply:
just a string comp whenever the error is returned via stderr from a runtime process